### PR TITLE
deps: enable automerge and set labels on renovate PRs

### PR DESCRIPTION
**Pull request by travisgroth**
## Summary
Streamline Renovate PR process.

**Checklist**:
- [x] ready for review

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,3 +1,11 @@
 {
-  "extends": ["config:base", "schedule:earlyMondays"]
-}
+  "extends": [
+    "config:base",
+    "schedule:earlyMondays"
+  ],
+  "automerge": true,
```
Comment (desimone) [inline renovate.json]: GRPC safe? 🤔

Comment (travisgroth) [reply renovate.json]: Won't merge without explicit approval since that's a status check that has to pass.  Once we hit approve, it'll go through.

**Comment by travisgroth** [comment]
>Does it wait till tests pass?

Yes all checks must pass, including a human approval. 🤖

**Comment by travisgroth** [comment]
>Does it wait till tests pass?

Yes all checks must pass, including a human approval. 🤖

File: renovate.json (hunk 1 above)
Comment (desimone) [reply renovate.json]: 👍

**Pull request merged**